pkg/kubectl/resource/helper.go

```diff
@@ -82,15 +92,26 @@ func (m *Helper) List(namespace, apiVersion string, selector labels.Selector, ex
 	return req.Do().Get()
 }
 
-func (m *Helper) Watch(namespace, resourceVersion, apiVersion string, labelSelector labels.Selector) (watch.Interface, error) {
+func (m *Helper) Watch(namespace, resourceVersion, apiVersion string, labelSelector labels.Selector, fieldSelector string) (watch.Interface, error) {
+	if labelSelector != nil && !labelSelector.Empty() && len(fieldSelector) > 0 {
```

Comment:
Not sure why you have this, they should be supported together.

Reply:
Right. I re-tested, the parser works for both. Seemed I had used a wrong example before.